UnityProject/Assets/Scripts/Objects/Doors/Modules/AccessModule.cs

```diff
@@ -89,10 +90,37 @@ private bool ProcessCheckAccess(GameObject player)
 			return false;
 		}
 
+		public bool ProcessCheckAccess(Access access)
+		{
+			if (accessRestrictions.restriction >= access)
+			{
+				return true;
+			}
+
+			//If the door is in low voltage, there's a very low chance the access check fails and opens anyway.
+			//Meant to represent the kind of weird flux state bits are when in low voltage systems.
+			if (master.Apc.State == PowerState.LowVoltage)
+			{
+				if (Random.value < lowVoltageOpenChance)
+				{
+					return true;
+				}
+			}
```

Comment:
Cool idea, but as it is I think it would easily be interpreted as a bug by players. An accommodating chat message might work.

Reply:
Not that I'm aware of. Hmmmm, "The airlock's control panel flickers for a moment..." 
Just something so that players can correlate the unexpected behaviour to something.

Reply:
The sad thing about this is that the station rarely ever goes under powered because our generators are too strong, and department batteries keep most of these things well powered for a long while. So the likelyhood of someone encountering this is very.. veeeeeeeeeery low.

Also done 👍

Reply:
Yeah. Electricity isn't quite right at the moment. Microwaves are overvolting-capable but we just can't do that yet, unfortunately : P